code/game/machinery/cloning.dm

```diff
@@ -524,6 +568,12 @@
 
 	flesh_number = unattached_flesh.len
 
+/obj/machinery/clonepod/mapped
+
+/obj/machinery/clonepod/mapped/Initialize()
+	. = ..()

```

Comment:
```suggestion
/obj/machinery/clonepod/prefilled

/obj/machinery/clonepod/prefilled/Initialize()
	. = ..()
```

Reply:
did that and it seems to work, even checked varedit in a test so its now that instead of /mapped